evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp

```diff
@@ -275,25 +275,25 @@ rcl_interfaces::msg::SetParametersResult PerceptionOnlineEvaluatorNode::onParame
 
   // update debug marker parameters
   {
-    const std::string ns = "debug_marker.";
+    const std::string name_space = "debug_marker.";
```

Comment:
@kobayu858 
Could you consider making line 250-274 to a code block by surrounding by `{}` ?

Like
```
// update parameters for each object class
{
  ...
}

// update debug marker parameters
{
  ...
}
```

Reply:
Thanks for the advice.
The warning has been fixed since it was confirmed that the warning was resolved by this method.